CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We are always welcome your contribution!
+
+## How to contribute
+
+### Find the tasks
+
+We use GitHub issues to manage the tasks. Please find the issues you'd like to contribute to it. `welcome contribute` and `easy` are good for the first contribution.
```

Comment:
Please check here for issues you would like to contribute to.

`welcome contribute` and `easy` are good labels to check for the first contribution.